### DISCUSSION TITLE
Cannot show Favorites window if only an epub file is opened.

### DISCUSSION THREAD

**Discussion by spinningwatt** [sumatrapdfreader/sumatrapdf#2452] [Q&A] [answered]
If I only open one epub file. I cannot right click to add favorite (CTRL+B still works). The favorite page has been added, but I cannot open the favorite window (there is no "Favorite" in menu).

However, then if I open any another pdf file, in pdf tab, I can open "Favorite" window and the favorite page in epub file also shows. But if I click the added favorite page, it cannot jump to the right page. "Favorite" seems doesn't work well with epub file.

 So, If I need to open the favorite window, I have to rely on an opened pdf file.

Is this a bug or is there any settings I missed?

**Comment by GitHubRulesOK** [comment, marked as answer]
Not a bug if using old flowing ebook User Interface (without zoom/find) since the favorites cannot in that case be static, but might work after a fashion, in some cases.
The current pre-release for version 3.4 uses the fixed user interface, more often for ebooks thus favorites will work as they are fixed page related.

**Reply by spinningwatt** [reply]
Thanks for your reply.

How can I get rid of the current flowing ebook User interface? I prefer the stable release.

**Reply by GitHubRulesOK** [reply]
in older versions change settings>advanced options 	`UseFixedPageUI = false` to true (however thats not available for new 3.4 pre-release so some formats are not now changed so can only change `ChmUI` to fixed
```
EbookUI [
	FontName = Georgia
	FontSize = 12.5
	TextColor = #5f4b32
	BackgroundColor = #fbf0d9
	UseFixedPageUI = true
]
```

**Reply by spinningwatt** [reply]
Thanks
